installlatest.sh

```diff
@@ -0,0 +1,58 @@
+
+get_latest_release() {
+	curl --silent "https://api.github.com/repos/$1/releases/latest" | grep -Po '"tag_name": "\K.*?(?=")'
```

Comment:
should there be some sort of sanity check here to make sure the request didn't fail etc.? (e.g. server could respond so curl "succeeded", but response wasn't in expected format and we get an invalid release tag etc.)

Reply:
well if the release tag isn't in a format available on the server, then the .tar.gz files downloaded won't be downloaded as they don't exist, which in return will not launch the installscript fully. As the paths and directories provided won't exist so after defining variables that script will exit as well. 
So errors for the .tar.gz files that it tries to find and then final error as variable path doesn't exist
![image](https://user-images.githubusercontent.com/51996872/94797713-6b1fd680-03d8-11eb-83bf-902b74627e9d.png)

Reply:
Sure, I just think it'd be cleaner if we detected it and exited the script instead of continuing with stuff that's going to fail anyway?

Reply:
Now looks for tag formatting, if it isn't in correct format, exits the script.